### PR TITLE
More release instructions

### PR DESCRIPTION
**Pull request by kierenevans**
A lot of what was being done wasn't covered in the instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,43 +92,84 @@ The [deploy script](./deploy) does a similar thing but the end result is output
...
+7. Create a "Github Release" for this repository and downstream repositories, pasting in the changelog for the release:
+   - https://github.com/inviqa/harness-base-php/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-php/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-akeneo/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-drupal8/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-magento1/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-magento2/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-spryker/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-symfony/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-wordpress/releases/new?tag=0.10.0
```
Comment (andytson-inviqa) [inline README.md]: You can actually create the tag from the new release page

Comment (kierenevans) [reply README.md]: Less typing if we push and then follow links.

Comment (andytson-inviqa) [reply README.md]: you still need to type to create a tag and push it, more so than following a link that can create a tag and release in one go

Comment (andytson-inviqa) [reply README.md]: https://github.com/inviqa/harness-wordpress/releases/new?tag=0.10.1&target=0.10.x

Comment (kierenevans) [reply README.md]: That's what the tag script does below, but the link you've provided is far better. Couldn't find that documented at github.

Will adjust.

File: README.md [hunk 2]
```diff
@@ -103,32 +103,72 @@ We are keeping a changelog, powered by [GitHub Changelog Generator].
...
+  docker run -e CHANGELOG_GITHUB_TOKEN="$CHANGELOG_GITHUB_TOKEN" -it --rm -v "$(pwd)":/usr/local/src/your-app -v "$(pwd)/github-changelog-http-cache":/tmp/github-changelog-http-cache ferrarimarco/github-changelog-generator --user inviqa --project harness-base-php --exclude-labels "duplicate,question,invalid,wontfix,skip-changelog" --release-branch 0.10.x --future-release 0.10.0
   ```
 4. Examine the generated CHANGELOG.md. For every entry in the `Merged pull requests` section, examine the Pull Requests
    and assign each pull request either a `enhancement` label for a new feature, `bug` for a bugfix or `deprecated` for
    a deprecation.
 5. For each Pull Request in the release, assign an appropriate `harness-*` label.
 6. Re-generate the changelog using step 3 as needed.
-7. Commit the resulting CHANGELOG.md, push and raise a pull request.
-8. Once merged, continue with the release process below.
+7. Adjust the version for each framework's README.md: `sed -i'' s/v0\.9\.0/v0.10.0/ src/*/README.md`
```
Comment (kierenevans) [inline README.md]:
```suggestion
7. Adjust the version for each framework's README.md: `sed -i'' s/v0\.9\.4/v0.10.0/ src/*/README.md src/*/docs/`
```

**Pull request merged**